lsp-mode.el

```diff
@@ -8216,7 +8221,10 @@ You may find the installation instructions at https://emacs-lsp.github.io/lsp-mo
        ;; no matches
        ((-> #'lsp--matching-clients? lsp--filter-clients not)
         (lsp--error "There are no language servers supporting current mode %s registered with `lsp-mode'."
-                    major-mode))))))
+                    major-mode)))
+      (unless (lsp--any-inconsistency?)
+        (lsp--warn "It's seems that one or more LSP packages are outdated or it may need to re-compile it to avoid errors.
```

Comment:
@yyoncho I'm not sure if this is the correct place to add the warning, could you please confirm that?